app/src/main/res/values-es/strings.xml

```diff
@@ -36,4 +35,14 @@
     <string name="all_time">Todo el tiempo</string>
     <string name="stats">Estadísticas</string>
     <string name="notification_channel_name">Notificaciones del rastreo</string>
+    <string name="import_file_item">Importar archivo</string>
```

Comment:
Do you actually speak Spannish? If not, just leave this file alone, I would say. (I do not, so can't comment on the strings here.)

Reply:
I did some research into other similar apps, so fairly confident these are accurate. But happy to drop them if you wish

Reply:
Nah, they can stay, I just wanted to avoid machine translation.